server/engine/src/test/java/org/eclipse/lsp/cobol/usecases/sql/TestSqlHostVariable.java

```diff
@@ -245,6 +245,42 @@ public class TestSqlHostVariable {
                   + "           DISPLAY {$VB-LENGTH}(1).\n"
                   + "           DISPLAY {$VB-DATA}(1).\n";
 
+  public static final String LOB_XML_ARR_TEXT1 =
+          "        Identification Division.\n"
+                  + "        Program-Id. 'TEST1'.\n"
+                  + "        Data Division.\n"
+                  + "         Working-Storage Section.\n"
+                  + "       01 {$*VAR-NAME1}.\n"
+                  + "       04 {$*VAR`->VAR`->VAR-LENGTH`->VAR-DATA} USAGE IS SQL TYPE IS XML AS CLOB (10) OCCURS 12345 TIMES.\n"
+                  + "        PROCEDURE DIVISION.\n";
+
+  public static final String LOB_XML_ARR_TEXT2 =
+          "        Identification Division.\n"
+                  + "        Program-Id. 'TEST1'.\n"
+                  + "        Data Division.\n"
+                  + "         Working-Storage Section.\n"
+                  + "       01 {$*VAR`->VAR`->VAR-LENGTH`->VAR-DATA} USAGE IS SQL TYPE IS XML AS CLOB(10) OCCURS 1234 TIMES.\n"
```

Comment:
```suggestion
                  + "       01 {$*VAR`->VAR`->VAR-LENGTH`->VAR-DATA|1} USAGE IS SQL TYPE IS XML AS CLOB(10) OCCURS 1234 TIMES.\n"
```
?

Reply:
This string is for a zero-error test, we should not be getting anything from diagnostics.

Reply:
Adding that will break the test. (Tried it locally first.) The point of this string is to test the locality/level not the variable.

Reply:
The level should not be 01, right?

Reply:
Correct, that is done on purpose. It is a check for when the user puts an invalid (in this instance) level on a XML file/reference variable array.